Exception subclass: #UnhandledError	instanceVariableNames: 'exception'	classVariableNames: ''	poolDictionaries: ''	category: 'Exceptions-Kernel'!!UnhandledError methodsFor: 'as yet unclassified' stamp: 'ajh 9/4/2002 19:15'!exception	^ exception! !!UnhandledError methodsFor: 'as yet unclassified' stamp: 'ajh 9/4/2002 19:15'!exception: anError	exception _ anError! !!UnhandledError methodsFor: 'priv handling' stamp: 'ar 9/27/2005 19:53'!defaultAction	"The current computation is terminated. The cause of the error should be logged or reported to the user. If the program is operating in an interactive debugging environment the computation should be suspended and the debugger activated."	^ToolSet debugError: exception.! !!UnhandledError methodsFor: 'priv handling' stamp: 'ajh 2/1/2003 00:56'!isResumable		^ false! !!UnhandledError class methodsFor: 'as yet unclassified' stamp: 'ajh 9/4/2002 19:17'!signalForException: anError	^ self new		exception: anError;		signal! !